src/coreclr/vm/peimagelayout.cpp

```diff
@@ -19,6 +19,23 @@ extern "C"
 }
 #endif
 
+namespace
+{
+    bool AllowR2RForImage(PEImage* pOwner)
```

Comment:
It could be a static method (without namespace). Tested with: https://github.com/am11/runtime/commit/a54f11be67733ce3fe78cc1908011233c8c98923?w=1

Reply:
Thanks.
* In mixed code-base C/C++, `static` is consistent, especially when there is no advantage of anonymous namespaces in play (scope level locals etc. like here).
* `static` is succinct in comparison and saves additional indentation due to added scope.